src/types/lineChart.ts

```diff
@@ -65,7 +63,7 @@ export interface LineChartData {
   /**
    * The fill color for the line chart.
    */
-  fill?: Color;
+  fill?: string;
```

Comment:
Same as above, to clarify string format assumed?
```suggestion
  /**
   * The html/hex color for the line chart fill.
   */
  fill?: string;
```